src/modules/character/index.js

```diff
@@ -227,5 +311,18 @@ router.get('/i/character/:id([0-9]+)/:region([A-Z]{2})/:realm([^/]{2,})/:name([^
   // noinspection JSIgnoredPromiseFromCall
   fetchCharacter(region, realm, name, !responded ? res : null);
 });
+router.get('/i/character/classic/:id([0-9]+)/:region([A-Z]{2})/:realm([^/]{2,})/:name([^/]{2,})', cors, async (req, res) => {
+  const { id, region, realm, name } = req.params;
+  const storedCharacter = await getStoredCharacter(id);
```

Comment:
making a note for later that the id space for classic may overlap with retail, but even if it does the odds of it ever mattering is pretty low